support-frontend/assets/pages/[countryGroupId]/events/events.tsx

```diff
@@ -0,0 +1,57 @@
+import { css } from '@emotion/react';
+import { from, sport } from '@guardian/source-foundations';
+import { Container } from '@guardian/source-react-components';
+import { FooterWithContents } from '@guardian/source-react-components-development-kitchen';
+import ophan from 'ophan';
+import { Header } from 'components/headers/simpleHeader/simpleHeader';
+import { PageScaffold } from 'components/page/pageScaffold';
+import ThankYouFooter from 'pages/supporter-plus-thank-you/components/thankYouFooter';
+
+const checkoutContainer = css`
+	${from.tablet} {
+		background-color: ${sport[800]};
+	}
+`;
+
+ophan.init();
```

Comment:
@paul-daniel-dempsey this is how I'm doing Ophan on events (related to checkout tracking).

Reply:
Thanks have put this into `assets/pages/[countryGroupId]/router.tsx` in my Ophan PR here -> (https://github.com/guardian/support-frontend/pull/6014)